dev/io.openliberty.microprofile.lra.coordinator.1.0.internal/resources/OSGI-INF/wlp/defaultInstances.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+    Copyright (c) 2020 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial implementation
+ -->
+<server>
+    <application context-root="/lrac" type="war" id="lra-c" location="${wlp.install.dir}/lib/mpLRACoordinator_5.10.5.jar"/>
```

Comment:
is `/larc` a standard context root? I presume that it can be customized by overriding this line in the server.xml?

Reply:
It was a random choice by me. I have an issue open to add config for this. I'll add a comment in the issue to say that a sensible default also needs to be chosen

Reply:
I suspect that is the kind of detail that should have been in the UFO? If it wasn't please update the UFO.